### ISSUE TITLE
Browser extension j2d fails in some cases on a GitHub PR

### ISSUE THREAD

**Issue by beyang** [sourcegraph/sourcegraph#1098]
- **Sourcegraph version:** sourcegraph.com
- **Platform information:** Chrome extension version 18.11.21.421
#### Steps to reproduce:

1. Go to https://github.com/sourcegraph/sourcegraph/pull/1026#discussion_r235062191
2. Click on the `enterprise/cmd/frontend/auth/githuboauth/config.go` file link
3. Arrive at https://github.com/sourcegraph/sourcegraph/pull/1026/files/bee9177b173fcbb25e8a0bdbdb40b87b8a83352e#diff-b5b7c1af05d00698e79e9b9487a0610f
4. Try to hover over tokens for code intelligence; observe that this doesn't work. JS console shows error:
```
Uncaught Error: File actions not found. Make sure you aren't trying to create a toolbar mount for a code snippet that shouldn't have one
```

After refreshing, hovers now work.

**Comment by nicksnyder** [comment]
You force pushed over that commit so I can't reproduce anymore.

**Comment by beyang** [comment]
Updated the repro with a new comment that works.

**Comment by felixfbecker** [comment]
Potentially fixed by #2909

**Comment by beyang** [comment]
Can't repro
